ioctl/output/format.go

```diff
@@ -21,6 +21,10 @@ type ErrorCode int
 const (
 	// UndefinedError used when an error cat't be classified
 	UndefinedError ErrorCode = iota
+	// UpdateError used when an error occurs when running update command
+	UpdateError
```

Comment:
Prefer to add errors to the end of the list for compatibility

Reply:
I have concerned about that, but UpdateError is special, so I put it at second place.